pkg/tagger/tagger.go

```diff
@@ -301,6 +303,18 @@ func (t *Tagger) Standard(entity string) ([]string, error) {
 	return t.tagStore.lookupStandard(entity)
 }
 
+// AgentTags returns the agent tags
+// It relies on the container provider utils to get the Agent container ID
+func (t *Tagger) AgentTags(cardinality collectors.TagCardinality) ([]string, error) {
```

Comment:
I'd suggest moving this implementation to the `AgentTags` func in `global.go`, to keep the tagger interface smaller, and we're about to introduce a second implementation, and I don't think this method would be any different between those two implementations.